build/Common.props

```diff
@@ -8,6 +8,8 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>net6.0;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
```

Comment:
```suggestion
    <DefaultTargetFrameworks>net6.0;netstandard2.1;netstandard2.0;net462</DefaultTargetFrameworks>
```

Reply:
You don't want to set all project's `TargetFrameworks` property because some projects (like the `OpenTelemetry.AotCompatibility.TestApp.csproj`) only want to target a single TFM. Using a different name here allows those other projects to not have multiple TFMs.